python/felis/db/variants.py

```diff
@@ -82,12 +140,12 @@ def make_variant_dict(column_obj: Column) -> dict[str, TypeEngine[Any]]:
 
     Parameters
     ----------
-    column_obj : `felis.datamodel.Column`
+    column_obj
         The column object from which to build the variant dictionary.
 
     Returns
     -------
-    variant_dict : `dict`
+    `dict` [ `str`, `~sqlalchemy.types.TypeEngine` ]
         The dictionary of `str` to `sqlalchemy.types.TypeEngine` containing
```

Comment:
To a certain extent you don't need to have the first half of this sentence given the dict types are specified in the line above. You can write it more generically as "The mapping of the variant database type (mysql, postgresql, etc) to the SQLAlchemy engine.".